stdlib/array.gr

```diff
@@ -145,6 +146,7 @@ export let rec init /*: (Number, Number -> a) -> Array<a>*/ =
  * @param index: The index to access
  * @param array: The array to access
  * @returns The element from the array
+ * @example Array.get([> 1, 2, 3, 4, 5]) == 2
```

Comment:
I think I messed up this suggestion. I think it needs 1 as the first argument

Reply:
@phated  Done. made the suggested changes

Reply:
Thanks! I'll kick off CI when I'm at my computer. Can you also make sure you regenerate the markdown docs so CI doesn't fail?

Reply:
I tried generating markdown and verifying it for the array section by running the command locally:
``` grain doc  stdlib/array.gr --current-version 0.5.3 > array.md ```
Is there any other better way of verifying the correctness md file?

Reply:
I think that command should work. Have you committed the changes?

Reply:
yes, committed & pushed

Reply:
I think you want to overwrite the stdlib/array.md file because you haven't pushed any markdown files in this PR

Reply:
Should we push the mardown file as well? i have array.md file did not existing in the repo.I generated it by running the above command.I don't think it has to be push, i feel, its generated dynamically.